src/config.rb

```diff
@@ -70,6 +72,18 @@ def configure_file
       File.join(repo_path, 'configure.yaml')
     end
 
-    attr_reader :cli
+    def domain_answers_file
```

Comment:
As above - I think it's clearer/ less likely to lead to bugs to have methods to get each of the different answer paths, but we can expose a shared method used by each of these for the validator if you like.